src/utils/components/SimpleCopyButton/SimpleCopyButton.scss

```diff
@@ -0,0 +1,5 @@
+#simple-copy-button__button {
+  background-color: white;
+  color: var(--pf-c-button--m-plain--Color);
+  padding-left: 5px;
```

Comment:
same as aboe, var for it? spacer xs or something

Reply:
Fixed!